.github/workflows/vpm_ci.yml

```diff
@@ -0,0 +1,60 @@
+name: VPM CI
+
+on:
+  push:
+    paths:
+      - '**/vpm_ci.yml'
+      - 'cmd/tools/vpm.yml'
+  pull_request:
+    paths:
+      - '**/vpm_ci.yml'
+      - 'cmd/tools/vpm.yml'
+
+concurrency:
+  group: vpm-ci-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  linux:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        cc: [tcc, gcc, clang]
+      fail-fast: false
+    env:
+      VFLAGS: -cc ${{ matrix.cc }} -d network
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build V
+        run: |
+          make -j4
+          [[ ${{ matrix.cc }} != "tcc" ]] && ./v -cc ${{ matrix.cc }} -cg -cstrict -o v cmd/v
```

Comment:
there is no need for that, `make` is enough

Reply:
Oh right, I would add a setup step then to build V once per os instead of re-build V for every matrix configuration.